default.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <remote name="origin" fetch="https://git.launchpad.net/~snappy-hwe-team/snappy-hwe-snaps/+git" />
+  <default revision="master" remote="origin"  sync-j="4" />
```

Comment:
That is a thing which we need to debate. "master" always reflects development. For documentation which reflects snap's released into the stable channel we need to use the "stable" branch of each repository which will always reflect the latest version either inside stable or candidate. However we should never use "master" here unless the whole documentation is explicitly flagged to represent development.